### DISCUSSION TITLE
Alterações Nota Técnica 2023.004 v.1.10

### DISCUSSION THREAD

**Discussion by Markosbn** [wmixvideo/nfe#992] [Geral]
Bom dia, gostaria de verificar se ja esta prevista as alteração ou se ainda não esta sendo feita da nota tecnica em questão para a biblioteca. Onde ha a adição de varios campos do grupo de pagamentos, YA

**Comment by fincatto** [comment]
Oi @Markosbn. 
Da minha parte, ainda não. Mas se você estiver precisando e tiver condições de implementar, seria ótimo!
Veja também se não entrou nos ultimos PRs que recebemos nas ultimas semanas.
Att

**Reply by walterportugal** [reply]
Todas as alterações de estrutura da NT 2023.004 já foram adicionadas. Eu fiz e estou inclusive utilizando.
